app/models/game.rb

```diff
@@ -112,18 +119,43 @@ def self.search(title, publisher, tourney, checked, loaned, group)
       result = result.includes(:loans).references(:loans).merge(Loan.where(closed: false, group: group, event: Event.current))
     end
 
-    result.where(culled: false)
+    result.where(status: 0)
   end
 
   def cull_game
     if self.checkouts.where(closed: false).size == 0
-      self.update(culled: true)
+      self.update(status: 1)
       'Game successfully culled!'
     else
       'Game is still checked out! Please return game before culling it.'
     end
   end
 
+  def toggle_storage_status
+    if self.checked_out?
+      return {
+        error: true,
+        message: 'Game is currently checked out by an attendee and can not be stored!'
+      }
+    end
+
+    if self.status == "storage"
```

Comment:
Prefer symbols to strings here, says Ruby best practices. Personally, I also think symbols look cleaner.

Reply:
Removed use of strings.